lawnchair/src/app/lawnchair/ui/preferences/about/About.kt

```diff
@@ -298,8 +298,29 @@ fun About(
                     }
                 },
             )
+            ClickablePreference(
+                label = stringResource(id = R.string.translate),
+                onClick = {
+                    val webpage = Uri.parse(CROWDIN_URL)
+                    val intent = Intent(Intent.ACTION_VIEW, webpage)
+                    if (intent.resolveActivity(context.packageManager) != null) {
+                        context.startActivity(intent)
+                    }
+                },
+            )
```

Comment:
Seems like you duplicated the "Translate" entry. Could you remove this first?

Reply:
Whoops, I meant to do R.string.donate

Reply:
Actually now fixed (removed)